CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 ## dbt-teradata 1.0.0a
 
 ### Features
+* dbt-teradata 1.6.7 is now compatible with dbt-core 1.6.7
 
 ### Fixes
-
+* Issues fixed related to cross DB macros ( https://github.com/Teradata/dbt-teradata/issues/105 , https://github.com/Teradata/dbt-teradata/issues/102)
+* Reconfiguration of dbt-teradata snapshot code for fixing (https://github.com/Teradata/dbt-teradata/issues/65)
```

Comment:
Please add more description about the issue. Its not clear what we fixed

Reply:
Added some more context